### PR TITLE
YLP-2300 Remove remote monitoring invitation

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/lib/medical-files/medical-files.api.ts [hunk 1]
```diff
@@ -25,29 +25,9 @@
...
 export default class MedicalFilesApi {
-  static async uploadPrescription(teamId: string, patientId: string, prescriptorId: string, initialPeriod: number, file: File): Promise<Prescription> {
-    const formData = new FormData()
-    formData.append('patientId', patientId)
-    formData.append('prescriptorId', prescriptorId)
-    formData.append('initialPeriod', initialPeriod.toString())
-    formData.append('teamId', teamId)
-    formData.append('upload', file)
-    const { data } = await HttpService.post<Prescription, FormData>({
-      url: 'cargo/v0/prescriptions',
```
Comment (fatb38) [inline packages/yourloops/lib/medical-files/medical-files.api.ts:40]: **❔ question:** ‏Is this need to be removed from Cargo too ?

Comment (TimCanu) [reply packages/yourloops/lib/medical-files/medical-files.api.ts:40]: It will have to eventually because it will be useless :slightly_smiling_face:

**Review by fatb38 — APPROVED** [review]

**Comment by TimCanu** [comment]
@fatb38 ticket already exists: https://diabeloop.atlassian.net/browse/YLP-2040

**Pull request merged**